src/history-api.ts

```diff
@@ -24,42 +24,65 @@ function convertMap(map: Map<string, browser.history.HistoryItem[]>) {
   return [...map.values()].map((day) => day.sort((a, b) => (b.lastVisitTime || 0) - (a.lastVisitTime || 0)));
 }
 
-function addEntries(daysMap: Map<string, browser.history.HistoryItem[]>, historyItems: browser.history.HistoryItem[]) {
+async function addEntries(
+  daysMap: Map<string, browser.history.HistoryItem[]>,
+  historyItems: browser.history.HistoryItem[],
+  repeatedVisits: boolean
+) {
   for (const historyItem of historyItems) {
-    daysMap.get(Moment(historyItem.lastVisitTime).format('YYYYMMDD'))!.push(historyItem);
```

Comment:
This can fail when viewing non-repeated visits for a previous day if the history item has last been visited today.

For example, if the current day is Sept. 10th and I have visited a website today, this cannot load visits for yesterday because `daysMap` contains a single key of `20240909`, so `daysMap.get(Moment(historyItem.lastVisitTime)` returns undefined.